emcc.py

```diff
@@ -219,10 +219,13 @@ class OFormat(Enum):
   JS = 2
   MJS = 3
   HTML = 4
+  BARE = 5
```

Comment:
I don't think it's obvious what this means, unlike the others.

Reply:
Yes I think we can certainly improve on the name. 

On reason I decided to include it in `--oformat` was that none of the other values of oformat make any sense when we stop after `wasm-ld` like this. 

For example if we invent a new flag call `--raw-wasm` then something like `--raw-wasm --oformat=js` would not have any meaning.